### ISSUE TITLE
Neonatal liver failure

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6185]
**Preferred term label:**
Neonatal liver failure

**Synonyms**
Neonatal hepatic failure

**Definition (free text, please give PubMed ID)**
Failure of the synthetic function of liver within 4 weeks of birth.
DOI: 10.1007/s00431-010-1309-1

**Parent term (use hpo.jax.org/app)**
Hepatic failure

**Comment by pnrobinson** [comment]
In this case we would use the term Hepatic failure (https://hpo.jax.org/app/browse/term/HP:0001399) together with the term Neonatal onset (https://hpo.jax.org/app/browse/term/HP:0003623). The HPO strives to avoid compound terms like this unless there are substantial phenotypic differences at different ages, I do not think this is the case with this item, but please reopen there are such differences.
